src/tests/Blef.GameLogic.Tests/PokerHandsHierarchy/PokerHandHierarchyTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Blef.GameLogic.PokerHands;
+using Blef.GameLogic.Tests.TestData;
+using NUnit.Framework;
+
+namespace Blef.GameLogic.Tests
+{
+    public class PokerHandHierarchyTest
+    {
+        [TestCaseSource(typeof(TestPokerHandsGenerator), nameof(TestPokerHandsGenerator.GeneratePokerHandsMatrix))]
+        public static void should_be_able_to_tell_which_one_pokerhand_is_stronger_than_another(int i, int j,
```

Comment:
I am afraid that this test is not going to find problem with Poker Hands Comparison like the bug #25. It is not testing any sofisticated edge cases.